tsconfig.json

```
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "declaration": true,
+    "esModuleInterop": true,
+    "isolatedModules": true,
+    "lib": [ "esnext" ],
+    "module": "commonjs",
+    "outDir": "build",
+    "resolveJsonModule": true,
+    "strict": true,
+    "target": "es2019"
+  },
+  "include": [
+    "./lib/**/*.ts",
+    "./test/**/*.ts",
+    "test/shared/dummyTs/dummyTsx.tsx"
```

To be consistent with the other paths:

```suggestion
    "./test/shared/dummyTs/dummyTsx.tsx"
```